app_usb_aud_xk_316_mc/src/extensions/audiohw.xc

```diff
@@ -479,6 +504,10 @@ void AudioHwConfig(unsigned samFreq, unsigned mClk, unsigned dsdMode, unsigned s
 
         SetI2CMux(PCA9540B_CTRL_CHAN_0);
     }
+    else if((XUA_SPDIF_RX_EN || XUA_ADAT_RX_EN) && USE_SW_PLL)
+    {
+        /* Do nothing - the SW_PLL configures the AppPLL */
+    }
     else
     {
         AppPllEnable(mClk);
```

Comment:
Hopefully this gets pulled into the previous statement with current work to use lib_sw_pll for fixed freqs?

Reply:
Yes